perllib/FixMyStreet/Script/Questionnaires.pm

```diff
@@ -45,20 +45,18 @@ sub send_questionnaires_period {
         $cobrand->set_lang_and_domain($row->lang, 1);
         FixMyStreet::Map::set_map_class($cobrand);
 
-        # Not all cobrands send questionnaires
-        next unless $cobrand->send_questionnaires;
-
-        # If cobrand doesn't allow users to
-        # update don't send questionnaires
-        next if $cobrand->deny_updates_by_user($row);
-
         # Cobrands can also override sending per row if they wish
         my $cobrand_send = $cobrand->call_hook('send_questionnaire', $row) // 1;
+        my $cobrand_deny = $cobrand->deny_updates_by_user($row);
+        my $no_reopen = $cobrand->reopening_disallowed($row, $row->user);
```

Comment:
No need for $row->user here, which dates from when I was trying to keep the user but not require `$c->user`, but then it turned out none of the user stuff was needed.